src/components/WalletConnect.vue

```diff
@@ -61,11 +58,23 @@ export default defineComponent({
 			});
 			if (userAccount && userAccount.length) {
 				this.walletAddress = userAccount[0].address;
+				this.text = "AlgoSigner";
 			}
 		},
-		handleMenuClick(e: Event) {
-			console.error("changing wallet kind", e);
+		handleMenuClick(e: any) {
+			console.error("changing wallet kind", e.key);
+			if (e.key === "1") {
```

Comment:
we must have an enum , something like `WalletKind.AlgoSigner`. I think @rmeena840  was adding it somewhere.